Os/Linux/FileSystem.cpp

```diff
@@ -317,19 +317,19 @@ Status initAndCheckFileStats(const char* filePath, struct stat* fileInfo = nullp
  * @param destination File to copy data to
  * @param size The number of bytes to copy
  */
-Status copyFileData(File source, File destination, FwSizeType size) {
```

Comment:
What is the reason for using a signed size here?

Reply:
It does look more consistent with the uses of signed sizes elsewhere. Is that the reason?

Reply:
I believe it was done to be consistent with the use of signed-sizes inherent to the Os::File class, which itself mirrors the use of `ssize_t` in the posix API.  I will come back to this in a moment once I have addressed the other comments.

Reply:
Yes, my belief is correct.  The File API deals in signed sizes to match the underlying calls (which reserve the right to set size to -1 on error).  This change has flowed out to all uses of Os::File.

Reply:
OK, sounds good.